src/it/xerces/src/test/java/com/jcabi/xerces/XercesSampleTest.java

```diff
@@ -56,6 +57,10 @@
      */
     @Test
     public void validatesXmlForSchemaValidity() throws Exception {
+        final int zero = 0;
```

Comment:
@longtimeago This http://www.qulice.com/quality.html says to avoid at all cost, so i'm going to leave this alone cause I don't want to change it back again later when @yegor256 object.